app/kubemci/pkg/gcp/healthcheck/healthchecksyncer.go

```diff
@@ -100,6 +91,37 @@ func (h *HealthCheckSyncer) EnsureHealthCheck(lbName string, ports []ingressbe.S
 	return ensuredHealthChecks, err
 }
 
+func getHealthCheckPath(port ingressbe.ServicePort, clients map[string]kubernetes.Interface) (string, error) {
+	var err error
+	path := ""
+	for cName, c := range clients {
+		probe, prErr := kubeutils.GetProbe(c, port)
+		if prErr != nil {
+			prErr = fmt.Errorf("Error %s when getting probe for port %v", prErr, port)
+			err = multierror.Append(err, prErr)
+			continue
+		}
+		if probe != nil && probe.HTTPGet != nil {
+			if path != "" {
+				// Ensure that all probes have the same health check path
+				if path != probe.HTTPGet.Path {
+					pathErr := fmt.Errorf("probe path '%s' from client '%s' does not match already extracted health check path '%s'", probe.HTTPGet.Path, cName, path)
+					err = multierror.Append(err, pathErr)
+					return path, err
+				}
+			} else {
+				// TODO use same method as in ingress-gce/backend to get probe settings, which is a private method currently
```

Comment:
Name the method here

Reply:
I'm thinking of implementing this as well before I'm done with the PR since that method gets more settings from the probe, like health check interval. Feels a bit half-baked if we go through all the trouble to get the probe and then only use the path.

Reply:
On second thought I think I'll leave that for later, it might become more straight forward once some refactoring has been done in ingress-gce and I should try to wrap up this PR.